### ISSUE TITLE
WAD Azure DevOps extention - Arguments not passed to selfhosted agent

### ISSUE THREAD

**Issue by maikelgithub** [microsoft/WinAppDriver#764]
Hi,

I am trying to make WAD work with our selfhosted agents. The setting I'm testing at the moment is passing the command line arguments to WAD.

My setup is this:
![image](https://user-images.githubusercontent.com/29131018/61376215-0c3c7300-a8a1-11e9-8fbf-a6d09a667ff6.png)

So I want to start WAD on port 7001.

These are the arguments passed through to the agent:

```
##[debug]Evaluating condition for step: 'Start - WinAppDriver'
##[debug]Evaluating: succeeded()
##[debug]Evaluating succeeded:
##[debug]=> True
##[debug]Result: True
##[section]Starting: Start - WinAppDriver
==============================================================================
Task         : WinAppDriver
Description  : Use this extention to integrate Windows Application Driver (WinAppDriver) into your Azure Pipeline.
Version      : 0.0.1
Author       : Microsoft
Help         : Checkout the WinAppDriver GitHub Repo to learn more - https://github.com/Microsoft/WinAppDriver
==============================================================================
##[debug]VstsTaskSdk 0.11.0 commit 7ff27a3e0bdd6f7b06690ae5f5b63cb84d0f23f4
##[debug]Entering D:\AgentAcc\_work\_tasks\Windows Application Driver_f7c56e2a-d1f1-4cff-b5d3-a32c9516c6e0\0.0.1\WADPowershell.ps1.
##[debug]Loading resource strings from: D:\AgentAcc\_work\_tasks\Windows Application Driver_f7c56e2a-d1f1-4cff-b5d3-a32c9516c6e0\0.0.1\task.json
##[debug]Loaded 0 strings.
##[debug]SYSTEM_CULTURE: 'en-US'
##[debug]INPUT_OPERATIONTYPE: 'Start'
##[debug]INPUT_WADARGUMENTS: '7001'
##[debug]INPUT_AGENTRESOLUTION: '4K'
Setting display resolution on Agent to - 4K
Launching WinAppDriver - no arguments.
WinAppDriver launch successful.

To close WinAppDriver & the console logs, please create another instance of this task following the Test Runner (VSTest or equivalent) 
in the pipeline and set the WinAppDriver task operation to "Close WinAppDriver"
##[section]Finishing: Start - WinAppDriver
```

**Issue:**
Even though the input argument is set with `INPUT_WADARGUMENTS: '7001'`, the argument is not passed to WAD. `Launching WinAppDriver - no arguments.` It wants to start on the default port (which in my case is already in use).

I'm assuming this has to be fixed in a update of the Azure DevOps WAD extention. If otherwise, I'm all ears.

**Comment by hassanuz** [comment]
Hey @maikelgithub, really appreciate you reaching out and informing me of this issue you're facing on your self-hosted agent. 

I just pushed out a new build of the WinAppDriver Pipelines Task to address - can you confirm that your version is now **0.0.2**, and try passing in the port as an argument now?

Let me know if this new build does the trick for you! 

If it still doesn't work, then there is a chance that you will have to disable Windows UAC on your self-hosted agent - though I wouldn't worry about that just now.

**Comment by maikelgithub** [comment]
hi @hassanuz 

thank you for reaching out and updating the WinAppDriver extention for Azure DevOps.

This new build is a succes for us, I tried various combinations and they all work!

And thank you for pointing out the Windows UAC control. We had to disable the "EnableLUA" option in the registry (\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System). After this change we were able to start and stop WinAppDriver in our environment.

Thank you once again. I will close this issue.
